scripts/content.js

```diff
@@ -0,0 +1,30 @@
+var novelData = [

```

Comment:
I think this whole page can be removed and the code in the other js file needs to be refactored with this page removed and the appropriate changes to link the json rather than this one. also i need to add my filters in here.